ios/Extensions.swift

```diff
@@ -210,7 +210,8 @@ extension UITableView {
         messageLabel.textAlignment = .center
         messageLabel.textColor = UIColor.RainbowTheme.Transactions.blueGreyDark35
         messageLabel.sizeToFit()
-        messageLabel.accessibilityIdentifier = "no-transactions-yet-label";
+        messageLabel.isAccessibilityElement = true
```

Comment:
You only need this if you're planning to tap on it

Reply:
test fails without on iOS 14, can be verified on CI with latest tests